api/editor/api-editor/src/main/java/ca/bc/gov/tno/areas/editor/models/ActionModel.java

```diff
@@ -54,10 +54,10 @@ public ActionModel() {
   }
 
   public ActionModel(Action entity) {
-    this(entity, "");
+    this(entity, "", entity.getVersion());
```

Comment:
The version is critical for updates to be successful.  Otherwise you get concurrency errors.  Fixed this in a number of places.